dqc_us_rules/resources/DQC_US_0015/dqc_15_exclusion_rules.csv

```diff
@@ -6,34 +6,29 @@ BLE,Member,,Contains_insensitive,Netting
 BLE,Member,,Equals,us-gaap:DeferredDerivativeGainLossMember
 BLE,Member,,Equals,us-gaap:AboveMarketLeasesMember
-BLE,Member,,Equals,us-gaap:MaterialReconcilingItemsMember
 BLE,Member,,Equals,us-gaap:NewAccountingPronouncementEarlyAdoptionEffectMember
 BLE,Member,,Equals,us-gaap:UnallocatedFinancingReceivablesMember
-BLE,Member,,Equals,us-gaap:AccumulatedNetGainLossFromDesignatedOrQualifyingCashFlowHedgesMember
-BLE,Member,,Equals,us-gaap:AccumulatedNetUnrealizedInvestmentGainLossMember
 BLE,Axis,,Equals,us-gaap:StatementEquityComponentsAxis
+BLE,Axis,,Has_member,dei,LegalEntityAxis|us-gaap:ParentCompanyMember
+BLE,Axis,,Has_member,dei,LegalEntityAxis|us-gaap:SubsidiariesMember
+BLE,Axis,,Has_member,dei,LegalEntityAxis|us-gaap:GuarantorSubsidiariesMember
+BLE,Axis,,Has_member,dei,LegalEntityAxis|us-gaap:NonGuarantorSubsidiariesMember
+BLE,Axis,,Has_member,dei,LegalEntityAxis|us-gaap:SubsidiaryIssuerMember
 BLE,Axis,,Equals,us-gaap:ErrorCorrectionsAndPriorPeriodAdjustmentsRestatementByRestatementPeriodAndAmountAxis
 BLE,Axis,,Equals,us-gaap:AdjustmentsForChangeInAccountingPrincipleAxis
 BLE,Axis,,Equals,us-gaap:AdjustmentsForNewAccountingPronouncementsAxis
 BLE,Axis,,Equals,us-gaap:ProspectiveAdoptionOfNewAccountingPronouncementsAxis
-BLE,Axis,,Equals,us-gaap:QuantifyingMisstatementInCurrentYearFinancialStatementsByNatureOfErrorAxis
-BLE,Axis,,Has_member,dei:LegalEntityAxis|us-gaap:ParentCompanyMember
-BLE,Axis,,Has_member,dei:LegalEntityAxis|us-gaap:SubsidiariesMember
-BLE,Axis,,Has_member,dei:LegalEntityAxis|us-gaap:GuarantorSubsidiariesMember
-BLE,Axis,,Has_member,dei:LegalEntityAxis|us-gaap:NonGuarantorSubsidiariesMember
-BLE,Axis,,Has_member,dei:LegalEntityAxis|us-gaap:SubsidiaryIssuerMember
-BLE,Axis,,Equals,us-gaap:ErrorCorrectionsAndPriorPeriodAdjustmentsRestatementByRestatementPeriodAndAmountAxis
-BLE,Axis,,Equals,us-gaap:AdjustmentsForChangeInAccountingPrincipleAxis
-BLE,Axis,,Equals,us-gaap:AdjustmentsForNewAccountingPronouncementsAxis
-BLE,Axis,,Equals,us-gaap:ProspectiveAdoptionOfNewAccountingPronouncementsAxis
-BLE,Axis,,Equals,us-gaap:QuantifyingMisstatementInCurrentYearFinancialStatementsByNatureOfErrorAxis
+BLE,Axis,,Equals,us-gaap:QuantifyingMisstatementInCurrentYearFinancialStatementsByNatureOfErrorAxi
 BLE,Axis,,Equals,us-gaap:PartnerCapitalComponentsAxis
 BLE,Axis,,Equals,us-gaap:ChangeInAccountingEstimateByTypeAxis
 BLE,Axis,,Equals,us-gaap:PartnerTypeOfPartnersCapitalAccountAxis
-BLE,Axis,,Has_member,us-gaap:StatementBusinessSegmentsAxis|us-gaap:CorporateAndOtherMember
-BLE,Axis,,Has_member,us-gaap:StatementBusinessSegmentsAxis|us-gaap:CorporateMember
-BLE,Axis,,Has_member,us-gaap:StatementBusinessSegmentsAxis|us-gaap:AllOtherSegmentsMember
+BLE,Member,,Has_member,us-gaap:StatementBusinessSegmentsAxis|us-gaap:CorporateAndOtherMember
+BLE,Member,,Has_member,us-gaap:StatementBusinessSegmentsAxis|us-gaap:CorporateMember
+BLE,Member,,Has_member,us-gaap:StatementBusinessSegmentsAxis|us-gaap:AllOtherSegmentsMember
```

Comment:
Can you update the Jira ticket with the additions/deletions made to the exclusion file, similar to what you did for the concepts file?